aea/aea_builder.py

```diff
@@ -780,7 +775,6 @@ def _set_component_build_directory(
             configuration.author,
             configuration.name,
         )
-        ensure_dir(configuration.build_directory)
```

Comment:
I think it should be kept here, but so:
```
if configuration.build_directory is not None:
    ensure_dir(configuration.build_directory)
```
This way, we don't require the package developer to do it!

Reply:
no, package developer has to do it.
so it should be implemented in build script.
this will help to avoid empty directories creation